install-host/install.sh

```diff
@@ -5,8 +5,12 @@ NC='\033[0m';
 
 ID=$(whoami);
 
-go() {
-	if command -v go &> /dev/null
+install_go() {
```

Comment:
the `go` name was detected by `command -v` and prevented installation.

Reply:
Yeah, sounds logical.